benchmarks/run_all.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# This script is invoked by benchmarking-platform shell scripts
+# to run all of the benchmarks defined in the tracer.
+
+set -ex
+
+for file in \
+  `dirname "$0"`/tracing_trace.rb \
+  `dirname "$0"`/gem_loading.rb \
+  `dirname "$0"`/profiler_*.rb;
+do
+  bundle exec ruby "$file"
+done
```

Comment:
So I don't think it's blocking for this PR, but I wonder if we should try to put in some kind of check for files that aren't run.

E.g. [something along the lines of this check](https://github.com/DataDog/dd-trace-rb/blob/master/spec/datadog/profiling/validate_benchmarks_spec.rb#L29)?

Otherwise, it can be a bit confusing that if anyone adds a new profiler benchmark, it gets automatically picked up, but if you add any other kind of benchmark, it doesn't?

Alternatively, should we just run every `.rb` file on the `benchmark/` folder, and move anything that's not a benchmark to a `lib/` subdirectory or something like that? (Or maybe list here the files that shouldn't be run instead of the ones that should?)

Reply:
I was actually not sure what the right approach to invoking "all" benchmarks was. Let me enumerate the files explicitly in this PR and try to improve the situation later.

Reply:
Move of supporting files to `lib` is actually part of https://github.com/DataDog/dd-trace-rb/pull/3787 which will be on top of this PR.

Reply:
Sounds reasonable -- now there's less magic, but it's a bit more consistent :+1: